http-client/src/main/java/io/airlift/http/client/HttpStatusListener.java

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Remove this license header, since this is a new class and the copyright is not correct